internal/driver/fetch.go

```diff
@@ -107,6 +137,36 @@ func fetchProfiles(s *source, o *plugin.Options) (*profile.Profile, error) {
 	return p, nil
 }
 
+func grabSourceAndBases(sources, bases []profileSource, fetch plugin.Fetcher, obj plugin.ObjTool, ui plugin.UI) (*profile.Profile, *profile.Profile, plugin.MappingSources, plugin.MappingSources, bool, int, int, error) {
```

Comment:
Can we avoid returning the profile counts and instead return an error if zero number of source profiles was fetched or if (non-zero number of base profiles was requested and zero was fetched)?